Solutions/Tomcat/Analytic Rules/TomcatMultipleEmptyRequestsFromSameIP.yaml

```diff
@@ -0,0 +1,34 @@
+id: 7c9a1026-4872-11ec-81d3-0242ac130003
+name: Tomcat - Multiple empty requests from same IP
+description: |
+  'Detects multiple empty requests from same IP'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: ApacheTomcat
+    dataTypes:
+      - TomcatEvent
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - InitialAccess
+  - Impact
+relevantTechniques:
+  - T1190
+  - T1133
+  - T1499
+query: |
+  let threshold = 50;
+  TomcatEvent
+  | where toint(HttpResponseBodyBytes) == 0

```

Comment:
Same here for toint() issue.

Reply:
Query works in Sentinel w/o casting toint().
Casting toint() was added to pass checks. If there is no casting toint() in files then checks fail with error